health-services/ExerciseSampleCompose/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+  dependencies {
+    classpath libs.kotlin.gradle.plugin
+    classpath 'com.google.dagger:hilt-android-gradle-plugin:2.43.2'
+
+    // NOTE: Do not place your application dependencies here; they belong
+    // in the individual module build.gradle files
+  }
+}
+
+plugins {
+  alias(libs.plugins.com.diffplug.spotless) apply(false)
+  alias(libs.plugins.com.android.application) apply(false)
+
+}
+
+subprojects {
```

Comment:
do not use the subprojects block as it's incompatible with isolated projects

Reply:
Done.